extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -371,6 +414,14 @@ class AmpLightbox extends AMP.BaseElement {
     this.triggerEvent_(LightboxEvents.CLOSE);
   }
 
+  /**
+   * @return {boolean}
+   * @private
+   */
+  isInabox_() {
```

Comment:
what do you think we make the close button a feature generic to amp-lightbox in both FIE or regular AMP. The feature is enabled by having element attribute "close-button". 

We then let the A4A validator to ensure its existence in ads.

Reply:
This is a great idea, and decreases complexity! Done.